doc/api/cli.md

```diff
@@ -1116,6 +1116,27 @@ unless either the `--pending-deprecation` command-line flag, or the
 are used to provide a kind of selective "early warning" mechanism that
 developers may leverage to detect deprecated API usage.
 
+### `--permission-fs-path-delimiter`
+
+<!-- YAML
+added: v20.0.0
```

Comment:
```suggestion
added: REPLACEME
```

Reply:
@Ceres6 you marked it as resolved, but it didn't change yet